packages/create-vite/template-vue-ts/src/components/HelloWorld.vue

```diff
@@ -25,7 +25,11 @@ const count = ref(0)
   </p>
   <p>
     Install
-    <a href="https://github.com/johnsoncodehk/volar" target="_blank">Volar</a>
+    <a
+      href="https://marketplace.visualstudio.com/items?itemName=Vue.volar"
```

Comment:
I think we should link to https://github.com/vuejs/language-tools instead? That way the user can access information about non-vscode integrations too

Reply:
Makes sense to me. Though there are several reference in the template to the volar link, should those be changed as well?

Reply:
I'd say we leave them be for now, and fix up the two lines in the PR only, since they used to point to the GitHub repo and we could maintain it.

Reply:
cool - updated